.github/workflows/ci.yml

```
@@ -92,6 +106,99 @@ jobs:
+        ports:
+          - 5432/tcp
+        options: >-
+          --health-cmd "pg_isready -U tldw_user -d tldw_content"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 10
+      redis:
+        image: redis:7-alpine
+        ports:
+          - 6379/tcp
+      redis:
```

YAMLlint (1.37.1)

[error] duplication of key "redis" in mapping

(key-duplicates)